recipes/simdjson/all/conanfile.py

```diff
@@ -49,10 +49,22 @@ def validate(self):
     def source(self):
         get(self, **self.conan_data["sources"][self.version], strip_root=True)
 
+    @property
+    def _cmake_cxx_standard(self):
+        compiler_cppstd = self.settings.get_safe("compiler.cppstd")
+        if compiler_cppstd is None:
```

Comment:
Note that in Conan 2 this should always be false

Reply:
@AbrilRBS I've copied this logic from conan https://github.com/conan-io/conan/blob/42c5c36383f0005db6e16614ddfe6fd1d9a26829/conan/tools/cmake/toolchain/blocks.py#L307C5-L317C52

Reply:
@Nekto89 Hello! Please, consider using the Recipe templates as reference instead: https://github.com/conan-io/conan-center-index/tree/master/docs/package_templates

Using internal Conan code is fragile as it may change, only public modules listed in the official documentation should be considered for compatibility.

Reply:
@uilianries I don't see this functionality in recipe templates.

Also I'm not calling internal conan code, I just copied the logic that to get C++ standard from compiler.cppstd I need to get it from settings, check for None and optionally remove "gnu" prefix. I understand that it may change in future but conan doesn't provide any other way to do it.